### DISCUSSION TITLE
Support for Dyson Light?

### DISCUSSION THREAD

**Discussion by NMBrux** [libdyson-wg/ha-dyson#199] [Ideas]
Hi, thanks a lot for the great work on this. 

I was wondering if it would be possible to add support for the solarcycle morph desk light: 

https://www.dyson.com/lighting/desk-lamps/solarcycle-morph-cd06/black-brass

Thanks in advance.

**Comment by dotvezz** [comment]
Hi @NMBrux! Unfortunately the Solarcycle lights don't have any Wi-Fi connectivity for device control, so they are not supported by this integration. 

For context, Solarcycle lights are Bluetooth devices controlled directly by the phone app.
